addons/custom-menu-bar/menu-labels.css

```diff
@@ -0,0 +1,3 @@
+[class*="menu-bar_file-group"] > [class*="menu-bar_hoverable"] > img {
+  display: none;
```

Comment:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/30805154-c98a-479a-84f7-dd1fcf3b14fb)

Could the margin or padding be reduced? When hovered they also look very slightly off-centred but I don't think that's a big deal.

Reply:
I can try...

Reply:
Is it better now?